.gitignore

```diff
@@ -1,3 +1,9 @@
 /docs/build/
 Manifest.toml
 .DS_Store
+
+/_readme/
```

Comment:
I guess that you do not actually want to ignore tracking of this full folder, right? We have there files that we want to track (e.g., README.jl).

Reply:
It was actually a `/_readme/Manifest.toml`, my bad!